server/src/api/modules/achievements/achievement.utils.ts

```diff
@@ -21,6 +22,9 @@ function getAchievementMeasure(metric: Metric, threshold: number): string {
 }
 
 function getAchievemenName(name: string, threshold: number): string {
+  if (name === 'Base {level} Stats') {
+    threshold = threshold / REAL_SKILLS.length;
```

Comment:
Could you do this same thing without assigning a new value to the parameter? maybe something like

```javascript
const adjustedThreshold = name === 'Base {level} Stats' ? threshold / REAL_SKILLS.length : threshold
```